msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -358,13 +365,10 @@ PUT {Service URL of your bot}/v3/conversations/{conversationId}/activities/{acti
 
 ## Get undelete message event
 
-When you undelete a message in a chat, the bot gets a notification of the undelete message event.
+When you undelete a message in the group chat or channel, the bot gets a notification of the undelete message event.
```

Comment:
update it to "When you undelete a message, the bot gets a notification of the undelete message **activity**."

Reply:
updated as suggested